dist/esm/modules/Toolbar.js

```diff
@@ -66,6 +68,7 @@ export class Toolbar extends BaseModule {
 			},
 			{
 				icon: IconAlignRight,
+				class: "ql-image-align-right",
```

Comment:
What's the benefit of the `ql-image-` prefix?

Reply:
I think `align-right` would be too generic and may clash easily. I followed the convention of quill who set `ql-align-right` and made it `ql-image-` for this package

Reply:
is the concern that `align-right` might be targetted by someone else's CSS ?

Reply:
I'll approve for now, but generally I advocate for semantic CSS names. I think the problems with styling clashes are often overstated.